config/virtual_models.yaml

```
@@ -389,8 +418,67 @@ providers:
+      daily_tokens: 500000
+      daily_requests: 14400
+    llama-3.3-70b-versatile:
+      tpm: 12000
+      daily_tokens: 100000
+      daily_requests: 1000
+    meta-llama/llama-4-scout-17b-16e-instruct:
+      tpm: 30000
+      daily_tokens: 500000
+      daily_requests: 1000
+
+  gemini:
```

YAMLlint (1.38.0)

[error] duplication of key "gemini" in mapping

(key-duplicates)